client/src/components/SearchResult/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import sitspotImg from '../../assets/searchResult';
+import './style.css';
+
+const SearchResult = ({ resultArray }) => {
+  let counter = 1;
+  return (
+    <ul className="list">
+      {resultArray.map(item => {
+        // this class to avoid margin in the third img in every row
+        let classNames = '';
```

Comment:
Up to you but maybe look into using the npm package `classnames` for this. It's an extra dependency yes but it does make the code for doing these things much more concise